aptos-move/framework/supra-framework/sources/multisig_account.move

```diff
@@ -768,6 +808,22 @@ module supra_framework::multisig_account {
         };
     }
 
+    /// Update the timeout duration for the multisig account.
+    entry fun update_timeout_duration(
+        multisig_account: &signer, timeout_duration: u64) acquires MultisigAccount {
+        assert_multisig_account_exists(address_of(multisig_account));
+        let multisig_account_resource = borrow_global_mut<MultisigAccount>(address_of(multisig_account));
+        let old_timeout_duration = multisig_account_resource.timeout_duration;
+        multisig_account_resource.timeout_duration = timeout_duration;
```

Comment:
assert that `timeout_duration >= MINIMAL_TIMEOUT_DURATION` before updating it.

Reply:
Done.